dev/com.ibm.websphere.appserver.features/visibility/public/jaxrs-2.2/com.ibm.websphere.appserver.jaxrs-2.2.feature

```diff
@@ -1,15 +1,15 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=com.ibm.websphere.appserver.jaxrs-3.0
+symbolicName=com.ibm.websphere.appserver.jaxrs-2.2
 visibility=public
 singleton=true
 IBM-API-Package: com.ibm.websphere.jaxrs.server; type="ibm-api", \
 IBM-App-ForceRestart: uninstall, \
  install
-IBM-ShortName: jaxrs-3.0
-Subsystem-Name: Java RESTful Services 3.0
+IBM-ShortName: jaxrs-2.2
+Subsystem-Name: Java RESTful Web Services 2.1 (RESTEasy)
```

Comment:
2.2

Reply:
The feature name doesn't normally mention the underlying implementation (RESTEasy).

Reply:
in this case we want it to - so that we can differentiate it from our normal/shipped version of the feature.  In this case, I really want 2.1 in the subsystem-name because it is implementing the 2.1 spec.  The transformed feature will be 3.0.  

@tevans78 for context, we are not planning to ship the 2.2 feature. This is where we will be pulling in RESTEasy and testing it, untransformed (i.e. in the `javax.ws.rs` package space). By having this feature, it makes it easier for the performance team to continue to benchmark CXF vs RESTEasy without needing to transform their app to the `jakarta.ws.rs` package space - and it allows us to test RESTEasy using our untransformed FATs.  The 3.0 feature will be transformed, but it will take some time before it is at the same point as the 2.2 (RESTEasy using `javax.*` APIs).